test/configCases/css/minimize-transforms-off/style.css

```
@@ -0,0 +1,28 @@
+/*! comments: at `"all"` an inert comment is kept, not just this banner */
+/* an inert comment */
+/*! shortenColors: the hex is not shortened */
+.t01 { color: #ffffff }
+/*! reduceFunctions: the calc is not folded and the transform names both axes */
+.t03 { width: calc(1px + 2px); transform: translate(1px, 0) }
+/*! shortenNumbers: the leading zero and the trailing fraction stay */
+.t05 { width: 0.50px; opacity: 100% }
+/*! normalizeQuotes: the string keeps the quote it was written with */
+.t06 { content: 'x' }
+/*! removeDeadRules: the second declaration of a property keeps the first */
+.t07 { color: red; color: red }
```

Stylelint (17.14.0)

[error] Duplicate property "color" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)